app/labor/bufferizer.rb

```diff
@@ -21,21 +21,22 @@ def satellite_tweet!
       if text.length < 250 && SiteConfig.twitter_hashtag
         text = text.gsub(" #{SiteConfig.twitter_hashtag}", " #{SiteConfig.twitter_hashtag} ##{tag.name}")
       end
-      BufferUpdate.buff!(@article.id, text, tag.buffer_profile_id_code, "twitter", tag.id, @admin_id)
+      BufferUpdate.buff!(@article.id, text, tag.buffer_profile_id_code, { tag_id: tag.id, admin_id: @admin_id })
```

Comment:
The `tag` object itself has been used twice at the same calling, it suggests to me that you could just pass the object itself for the `buff!` method.